### ISSUE TITLE
Slower seek performance in flatpak compared to apt version

### ISSUE THREAD

**Issue by ComplexPlane** [flathub/io.mpv.Mpv#111]
On Ubuntu 22.04, mpv in flatpak form has slower seek performance than the apt package. Both are currently v0.34.1. While playing a video file, holding right stutters immensely on the flatpak mpv, but is very snappy on the apt version.

No idea if this is some flatpak limitation, I hope not! I'd like to think flatpaks are capable of nearly identical I/O perf.

**Comment by fastrizwaan** [comment]
Yes, intel drivers on wayland has issues, https://wiki.archlinux.org/title/mpv#Fix_jerky_playback_and_tearing

copy /etc/mpv/mpv.conf to ~/.var/app/io.mpv.Mpv/config/mpv/mpv.conf

```
cat ~/.var/app/io.mpv.Mpv/config/mpv/mpv.conf 
hwdec=auto-safe
vo=xv
```
I'll add hwdec=auto to mpv.conf in flatpak.
